CHANGES.rst

```diff
@@ -1,9 +1,10 @@
 Changes
 =======
 
-Version 0.8.2 (UNRELEASED)
+Version 0.9.0 (UNRELEASED)
```

Comment:
You can keep the 0.8.2 line that we might release for 0.8 bug fixes.

So just add a new 0.9.0 section for the current change.